server/src/diskdb.ts

```diff
@@ -0,0 +1,33 @@
+import { Result, Database, DatabaseType } from '../../frontend/src/types';
```

Comment:
I will implement this disk db next as discussed and update the PR for final review :)